src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  bookList: [],
+};
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    add: (state, action) => {
+      const {title, author} = action.payload;
+      const newBook = {
+        category: 'Undefined',
+        title: title,
+        author: author,
+      }
+      return{
+        ...state,
+        bookList: [...state.bookList, newBook]
+      }
+    },
+    remove: (state, action) => {
+      const updatedList = state.bookList.filter((book, index) => {
+        index !== action.payload
+      });
```

Comment:
- [OPTIONAL] Consider filtering books based on their `id` property instead of the `index`. This provides a more reliable and accurate way to locate specific books for manipulation.
```suggestion
    remove: (state, action) => {
      const updatedList = state.bookList.filter((book) => {
        book.id !== action.payload
      });
```